src/main/java/frc/robot/RobotContainer.java

```diff
@@ -70,7 +72,7 @@ public RobotContainer() {
     tankSubsystem = new TankSubsystem();
 
     // Instantiate commands
-    tankCommand = new DriveTankCommand(tankSubsystem, 0, 0);
+    tankCommand = new DriveTankCommand(tankSubsystem, 12);

```

Comment:
Is this "12" related to the "12 inches" comment on L117?
I don't love magic numbers, but a comment here would be helpful if you don't want to make this a well-named constant.

Reply:
Yeah, the idea was for it to drive forward "12 inches" in autonomous to then measure that distance and check if the constant was working properly, but the different speeds on each side prevented that from really happening. I'll add a comment to this.